src/appdev/ApplicationUtils.js

```diff
@@ -32,7 +32,14 @@ const insertIntoMySQLStatement = (
   return `INSERT INTO ${tableName} ${columns} VALUES ${values};`;
 };
 
+function getESTDateString() {
+  let usaTime = new Date().toLocaleString('en-US', { timeZone: 'America/New_York' });
+  usaTime = new Date(usaTime);
+  return usaTime.toLocaleString();
```

Comment:
wait i don't understand why you have to do this twice? you make the date, make it a string, make it a date, and make it a string again?

Reply:
the first line gets the string, and the second line creates a date object

Reply:
right but why do you need lines 2 and 3? the first one makes the date and makes it a string!